Library/Homebrew/cmd/tests.rb

```diff
@@ -1,9 +1,34 @@
 require "fileutils"
+require "official_taps"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+    Homebrew.install_gem_setup_path! "bundler"
+
+    if ARGV.include? "--official-cmd-taps"
+      OFFICIAL_CMD_TAPS.each do |org, repo|
+        tap = "#{org}/#{repo}"
+        tap_dir = HOMEBREW_LIBRARY/"Taps/#{org}/homebrew-#{repo}"
+
+        # It has no tests :(
+        next if "#{tap}" == "homebrew/services"
+        ohai "Running #{tap} tests:"
+
+        system "brew", "tap", tap unless tap_dir.exist?
```

Comment:
If we stick with the `system` version, use `HOMEBREW_BREW_FILE` instead of `"brew"` like we do in most other places like [`Library/brew.rb`](https://github.com/Homebrew/brew/blob/915eed4c6427271c5b4c01fa36239361c3680c40/Library/brew.rb#L112-L113).